src/useInterval.tsx

```diff
@@ -0,0 +1,28 @@
+import {useRef, useEffect} from 'react'
+
+// https://overreacted.io/making-setinterval-declarative-with-react-hooks/
+const useInterval = (
```

Comment:
I didn't expose `useInterval()`, just used internally for implementing `addInterval()`

But there is an interesting question where a lot of these helpers could be implement as custom vanilla hooks + a thin ad-hok wrapper, so this library could expose corresponding custom vanilla hooks

But that would be weird conceptually/dependency-wise for people to use an ad-hok-utils library when not using ad-hok

So if that were worth doing, maybe that'd be a separate vanilla-hooks-only package that this package would then depend on?